jac-uk/apply
# #1044 Add Guidance Instructions

## What's included?
Added instructions above Apply button.

Closes #1044 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Click on any vacancy.
- Ensure there is a warning above the Apply button that says:

'By clicking the apply button you are agreeing that you have read all the information on the exercise information page and any guidance referred to.'

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context

<img width="1728" alt="Screenshot 2023-08-18 at 13 26 20" src="https://github.com/jac-uk/apply/assets/115651787/aa6c2759-73d4-44e3-a749-f5bcc291a1c0">

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
